README.md

```diff
@@ -87,91 +108,80 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
    ${FLY_CMD} -t ${DEPLOY_ENV} login -k -c "https://${DEPLOY_ENV}-concourse.cf.paas.alphagov.co.uk"
 ```
 
-## BOSH and Cloud Foundry
+### Destroy
 
-### Microbosh deployment from concourse bootstrap
+Run the `destroy-deployer` pipeline from your *Bootstrap Concourse*.
 
-These pipelines will deploy/destroy a microbosh using bosh-init. It will use the IAM profile `deployer-concourse` that was given to the Concourse instance.
```

Comment:
I'd be nervous about linking to a doc for this as it's bound to be out of date. Maybe it would be enough to link to the terraform repo that specifies all of these.